1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,153 @@
+Feature: User edits Project
+# First three lines cover Why, Who, and What, following scenarios covers the How
+In order to document a project 
+As a logged in User or as a Curator
+I want to edit existing and create new projects
```

Comment:
```suggestion
# In order to document a project 
# As a logged in User
# I want to edit existing and create new projects
```

Reply:
Usikker på om desse tre linjene trenger å vere kommentarar - ref:
https://cucumber.io/docs/bdd/who-does-what/#the-three-amigos/

Reply:
det har du nok rett i. har bare ikke sett det der før